frontend/test/reducers/reducers.spec.js

```diff
@@ -13,4 +18,57 @@ describe('channelPlatformSummary reducer', () => {
 
     expect(expectedState).toEqual(receivedState);
   });
+
+  it('Receives channel platform summary data', () => {
```

Comment:
Could you add a test for `REQUEST_CHANNEL_PLATFORM_SUMMARY_DATA` here, sort of parallel to what you did below with `REQUEST_MEASURE_DATA`?